osquery/tables/applications/darwin/browser_plugins.cpp

```diff
@@ -76,6 +76,14 @@ void genBrowserPlugin(const std::string& uid,
   }
 
   r["path"] = path;
+
+  // Check to see if the plugin 'lives' in the disabled folder.
+  if(path.find("Disabled") != std::string::npos){
```

Comment:
What if the plugin's name is something like: "Highlight Disabled TextFields"? This will catch some false positives, better to assure you're enumerating from within the Disabled folder.